app/build.gradle

```diff
@@ -18,6 +18,9 @@ android {
     }
 
     buildTypes {
+        debug {
+            buildConfigField 'String', "API_KEY", API_KEY
```

Comment:
you should have here the key "\"027b7152f20f23f8f0cbc9f4bfab880c\""

Reply:
Solved!